ui/raidboss/data/02-arr/raid/t5.ts

```diff
@@ -1,14 +1,58 @@
+import Conditions from '../../../../../resources/conditions';
 import NetRegexes from '../../../../../resources/netregexes';
-import Regexes from '../../../../../resources/regexes';
 import { Responses } from '../../../../../resources/responses';
+import Util from '../../../../../resources/util';
 import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
 
-export default {
+export interface Data extends RaidbossData {
+  monitoringHP: boolean;
+  hpThresholds: number[];
+  currentPhase: number;
+}
+
+const triggerSet: TriggerSet<Data> = {
   zoneId: ZoneId.TheBindingCoilOfBahamutTurn5,
   timelineFile: 't5.txt',
+  initData: () => {
+    return {
+      monitoringHP: false,
+      hpThresholds: [0.85, 0.55, 0.29],
+      currentPhase: 0,
+    };
+  },
   triggers: [
+    {
+      id: 'T5 Twintania Phase Change Watcher',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ source: 'Twintania' }),
```

Comment:
I'm a little bit hesitant to have a netregex that just matches all abilities, but doing it this way does save writing the trigger three times.  This is probably the best way to do it.